controllers/fio/job.go

```diff
@@ -0,0 +1,84 @@
+/*
+Copyright 2019 The xridge kubestone contributors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package fio
+
+import (
+	batchv1 "k8s.io/api/batch/v1"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	"github.com/firepear/qsplit"
+	perfv1alpha1 "github.com/xridge/kubestone/api/v1alpha1"
+)
+
+// NewJob creates a fio benchmark job
+func NewJob(cr *perfv1alpha1.Fio) *batchv1.Job {
```

Comment:
Can you please create a quick test for this function?

Reply:
created